code/datums/wires/wires.dm

```diff
@@ -27,7 +26,12 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 	..()
 	src.holder = holder
 	if(!istype(holder, holder_type))
+#ifdef DISABLE_DEBUG_CRASH
+		to_world_log("Our holder is null/the wrong type!") 
```

Comment:
CRASH_WITH supplies debugging info (stack trace), this just prints a sourceless message without saying who 'our' refers to.

Reply:
we're pro engineers. we don't need stack traces. honestly I was tempted to remove the to_world_log entirely.

Reply:
That would be preferrable to the current line.